### PR TITLE
Fixed Payments table, but we still need to fix inserts(

### PR DESCRIPTION
**Pull request by Alexenon**
Guys we need to fix inserted rows in the Payments table, columns `payment_date` and `id_subsidiary`

### CODE REVIEW

File: Coffee_Time.sql [hunk 1]
```diff
@@ -78,7 +78,8 @@ CREATE TABLE Products(
 CREATE TABLE Payments(
 	id_payment INT PRIMARY KEY,
 	payment_type NVARCHAR(4) CHECK(payment_type='CASH' OR payment_type='CARD') NOT NULL,
-    id_schedule INT FOREIGN KEY REFERENCES Employers_Schedule(id_Schedule)
+	payment_date DATETIME NOT NULL,
+    id_subsidiary INT FOREIGN KEY REFERENCES Subsidiaries(id_Subsidiary)
```
Comment (vldzz) [inline Coffee_Time.sql:82]: Add one more tab

**Review by vldzz — CHANGES_REQUESTED** [review]
Inserts need to be  fixed

File: Coffee_Time.sql (hunk 1 above)
Comment (Alexenon) [reply Coffee_Time.sql:82]: What do you mean ?

Comment (vldzz) [reply Coffee_Time.sql:82]: Should be at the same level
image

Comment (Alexenon) [reply Coffee_Time.sql:82]: They are at same level
image

**Review by vldzz — APPROVED** [review]
Approved

**Pull request merged**